2021-03-10 23:44:30,133 INFO    MainThread:11100 [wandb_setup.py:_flush():69] setting env: {}
2021-03-10 23:44:30,133 INFO    MainThread:11100 [wandb_setup.py:_flush():69] setting login settings: {}
2021-03-10 23:44:30,134 INFO    MainThread:11100 [wandb_init.py:_log_setup():334] Logging user logs to C:\Users\rao_a\PythonCPU\Deep Learning\CS6910-Assignments-1\Assignment1\Avyay\wandb\run-20210310_234430-3de70mu3\logs\debug.log
2021-03-10 23:44:30,134 INFO    MainThread:11100 [wandb_init.py:_log_setup():335] Logging internal logs to C:\Users\rao_a\PythonCPU\Deep Learning\CS6910-Assignments-1\Assignment1\Avyay\wandb\run-20210310_234430-3de70mu3\logs\debug-internal.log
2021-03-10 23:44:30,135 INFO    MainThread:11100 [wandb_init.py:_jupyter_setup():285] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000016778F01048>
2021-03-10 23:44:30,135 INFO    MainThread:11100 [wandb_init.py:init():367] calling init triggers
2021-03-10 23:44:30,135 INFO    MainThread:11100 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-03-10 23:44:30,135 INFO    MainThread:11100 [wandb_init.py:init():416] starting backend
2021-03-10 23:44:30,136 INFO    MainThread:11100 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=spawn, using: spawn
2021-03-10 23:44:30,140 INFO    MainThread:11100 [backend.py:ensure_launched():123] starting backend process...
2021-03-10 23:44:30,289 INFO    MainThread:11100 [backend.py:ensure_launched():128] started backend process with pid: 13824
2021-03-10 23:44:30,290 INFO    MainThread:11100 [wandb_init.py:init():421] backend started and connected
2021-03-10 23:44:30,307 INFO    MainThread:11100 [wandb_init.py:init():461] updated telemetry
2021-03-10 23:44:30,308 INFO    MainThread:11100 [wandb_init.py:init():480] communicating current version
2021-03-10 23:44:33,093 INFO    MainThread:11100 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.10.22 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-03-10 23:44:33,094 INFO    MainThread:11100 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-03-10 23:44:33,749 INFO    MainThread:11100 [wandb_init.py:init():518] starting run threads in backend
2021-03-10 23:44:35,356 INFO    MainThread:11100 [wandb_run.py:_console_start():1476] atexit reg
2021-03-10 23:44:35,357 INFO    MainThread:11100 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.WRAP
2021-03-10 23:44:35,358 INFO    MainThread:11100 [wandb_run.py:_redirect():1374] Wrapping output streams.
2021-03-10 23:44:35,358 INFO    MainThread:11100 [wandb_run.py:_redirect():1390] Redirects installed.
2021-03-10 23:44:35,358 INFO    MainThread:11100 [wandb_init.py:init():542] run started, returning control to user process
2021-03-10 23:44:35,363 INFO    MainThread:11100 [wandb_init.py:_pause_backend():247] pausing backend
2021-03-10 23:44:35,363 INFO    MainThread:11100 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-03-10 23:46:33,547 INFO    MainThread:11100 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2021-03-10 23:46:33,548 INFO    MainThread:11100 [wandb_run.py:_restore():1418] restore
2021-03-10 23:46:33,882 INFO    MainThread:11100 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 921
  total_bytes: 921
}

2021-03-10 23:46:35,884 INFO    MainThread:11100 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4411
  total_bytes: 4411
}

2021-03-10 23:46:37,885 INFO    MainThread:11100 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4411
  total_bytes: 4411
}

